pretix_attestation_plugin/views.py

```diff
@@ -24,24 +21,19 @@ def get_form_kwargs(self):
         return kwargs
 
     def form_valid(self, form):
-        self.write_to_file(form.cleaned_data["key_file_data"])
+        self.write_to_file(self.request.FILES.get('keyfile', None))
```

Comment:
you still want to get the file data from the `form.cleaned_data` - the form picks it up from the right places from request - well, as long as the form is properly configured!